api/server/sdk/server.go

```diff
@@ -343,9 +348,12 @@ func newSdkGrpcServer(config *ServerConfig) (*sdkGrpcServer, error) {
 		name:            name,
 		log:             log,
 		clusterHandler:  config.Cluster,
-		driverHandler:   d,
-		alertHandler:    config.AlertsFilterDeleter,
-		policyServer:    config.StoragePolicy,
+		driverHandlers: map[string]volume.VolumeDriver{
```

Comment:
I'm not sure I understand this. A `map[string]volume.VolumeDriver` would mean that every value would have a DefaultDriverName. What if two values have two different defaultdrivernames?

Is something like this what you meant?:

```go
type VolumeDrivers struct {
  Drivers []string
  DefaultDriverName string
}
...
driverHandlers *VolumeDrivers,
...
```

Reply:
Or is it that `DefaultDriverName` is a special key where its value is the default driver?

Reply:
We need a map[string] VolumeDrivers, because the sdk client request is going to pass in a string which indicates which VolumeDriver to use.

DefaultDriver is required so that if the context does not have any requested drivername provided, then we will use the default one which is registered.

Reply:
I get it now, thanks!